CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+NEXT_RELEASE Release notes (YYYY-MM-DD)
+=============================================================
+
+### Fixed
+* None
+
+### Enhancements
+* Add `realm::db_config::enable_forced_sync_history()` which allows you to open a synced Realm
```

Comment:
Can you add a little bit of context why do we need this flag?, I supposed if we are offline and cannot log the user we can make use of this, but in that case there is a user in the cache that can be used to open the sync realm offline.

Reply:
The purpose is for when are logged out due to the refresh token expiration, and due to connectivity issues you cannot log back in.

Reply:
Is this available in any other SDKs?, or Is this a request for a specific customer?

Reply:
I didn't see it in the Swift SDK, this is for a specific customer requirement.